content/html/concepts/tags/terms/center/center.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: '<center>'
+Description: 'Displays its contents centered horizontally in the containing element. This is now deprecated and CSS should be used instead.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Tags'
+  - 'Style'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<center>` tag was used historically to center content, usually text, within its containing elements. It has since been deprecated and removed from the HTML specification. While some browsers might support it for backwards compatibility, it should no longer be used.
+
+The current practice in HTML is to use the [CSS](https://www.codecademy.com/resources/docs/css) property [`text-align`](https://www.codecademy.com/resources/docs/css/typography/text-align) to center text and to use properties of the [CSS box model](https://www.codecademy.com/resources/docs/css/box-model) to center other elements.
+
```

Comment:
Add syntax section here. And perhaps mention it's usually found within the `<body>` container tag. I think that would be helpful for those who may be trying to look for it in older documents.